### PR TITLE
Add GoCD Units

### PR DESCRIPTION
**Pull request by hazim1093**

### CODE REVIEW

File: resources/cloud-config/admiral.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #cloud-config
 
-coreos: 
+coreos:
   etcd2:
-    proxy: on
+    discovery: https://discovery.etcd.io/59b91eba6a0705e55ff470943ee5c14a
```
Comment (rasheedamir) [inline resources/cloud-config/admiral.yaml:5]: @hazim1093 you confirmed we can use the same? or we shld create a ticket that we need to handle to create it dynamically and unique each time?

Comment (hazim1093) [inline resources/cloud-config/admiral.yaml:5]: We can't use the same for different instances, we need to create it dynamically.

Comment (rasheedamir) [inline resources/cloud-config/admiral.yaml:5]: so, will you plz create an ticket on which we can work later?

Comment (hazim1093) [inline resources/cloud-config/admiral.yaml:5]: Created https://github.com/stakater/stakater/issues/26

**Pull request merged**